packages/features/ee/organizations/lib/onboardingStore.ts

```diff
@@ -117,6 +121,7 @@ export const useOnboarding = (params?: { step?: "start" | "status" | null }) =>
   const router = useRouter();
   const path = usePathname();
   const isAdmin = session.data?.user?.role === UserPermissionRole.ADMIN;
+  const isBillingEnabled = !(IS_SELF_HOSTED && isAdmin) || process.env.NEXT_PUBLIC_IS_E2E;
```

Comment:
Shouldn't we reuse this, in case someone wants to enable organizations self serve in their self-hosted instance.
https://github.com/calcom/cal.com/blob/self-hosting-org-creation/packages/lib/constants.ts#L119